website/content/docs/nia/terraform-modules.mdx

```diff
@@ -154,14 +154,12 @@ To configure a task with the Consul KV source input, the KVs which will be used
 - a [`condition "consul-kv"` block](/docs/nia/configuration#consul-kv-condition) configured with the `source_includes_var` set to true.
 - a [`source_input "consul-kv"` block](/docs/nia/configuration#consul-kv-source-input).
 
-Below is a similar example to the one provided in the [Consul KV Condition](/docs/nia/tasks#consul-kv-condition) section. However, the difference in this example is that instead of triggering based on a change to Consul KV, this task will instead execute on a schedule. Once execution is triggered, Consul KV information is then provided to the task’s module.
+Below is a similar example to the one provided in the [Consul KV Condition](/docs/nia/tasks#consul-kv-condition) section. However, the difference in this example is that instead of triggering based on a change to Consul KV, this task will instead execute on a schedule. Once execution is triggered, Consul KV information is then provided to the task's module.
 
 ```hcl
 task {
   name        = "consul_kv_schedule_task"
   description = "executes on Monday monitoring Consul KV"
-  providers   = ["my-provider"]
```

Comment:
Just double checking you meant to remove the provider

Reply:
Yes, sorry, I should have added a comment explaining this. I thought `provider` wasn't necessary to this example and wanted to trim it down a bit.